chapters/descriptorsets.txt

```diff
@@ -2190,7 +2307,32 @@ The allocated descriptor sets are returned in pname:pDescriptorSets.
 
 When a descriptor set is allocated, the initial state is largely
 uninitialized and all descriptors are undefined:.
-Descriptors also become undefined: if the underlying resource is destroyed.
+
+Descriptors also become undefined: when:
+
+ifdef::VK_VALVE_mutable_descriptor_type[]
+  * Updating a descriptor of ename:VK_DESCRIPTOR_TYPE_MUTABLE_VALVE type,
```

Comment:
Does it make more sense to make some of these VUs instead of having them make the descriptor undefined. That is a tiny bit more restrictive but would make it likely massively easier to test the added bits in the validation layers?

Reply:
I guess this can be a VU.